lib/rucio/core/transfer.py

```diff
@@ -849,60 +850,61 @@ def update_transfer_state(external_host, transfer_id, state, session=None, logge
-            reqs = request_core.get_requests_by_transfer(external_host, transfer_id, session=session)
-            for req in reqs:
-                logger(logging.INFO, 'REQUEST %s OF TRANSFER %s ON %s STATE %s' % (str(req['request_id']), external_host, transfer_id, str(state)))
-                src_rse_id = req.get('source_rse_id', None)
-                dst_rse_id = req.get('dest_rse_id', None)
-                src_rse = None
-                dst_rse = None
-                if src_rse_id:
-                    src_rse = get_rse_name(src_rse_id, session=session)
-                if dst_rse_id:
-                    dst_rse = get_rse_name(dst_rse_id, session=session)
-                response = {'new_state': state,
-                            'transfer_id': transfer_id,
-                            'job_state': state,
-                            'src_url': None,
-                            'dst_url': req['dest_url'],
-                            'duration': 0,
-                            'reason': "The FTS job lost",
-                            'scope': req.get('scope', None),
-                            'name': req.get('name', None),
-                            'src_rse': src_rse,
-                            'dst_rse': dst_rse,
-                            'request_id': req.get('request_id', None),
-                            'activity': req.get('activity', None),
-                            'src_rse_id': req.get('source_rse_id', None),
-                            'dst_rse_id': req.get('dest_rse_id', None),
-                            'previous_attempt_id': req.get('previous_attempt_id', None),
-                            'adler32': req.get('adler32', None),
-                            'md5': req.get('md5', None),
-                            'filesize': req.get('filesize', None),
-                            'external_host': external_host,
-                            'job_m_replica': None,
-                            'created_at': req.get('created_at', None),
-                            'submitted_at': req.get('submitted_at', None),
-                            'details': None,
-                            'account': req.get('account', None)}
-
-                err_msg = request_core.get_transfer_error(response['new_state'], response['reason'] if 'reason' in response else None)
-                request_core.set_request_state(req['request_id'],
-                                               response['new_state'],
-                                               transfer_id=transfer_id,
-                                               src_rse_id=src_rse_id,
-                                               err_msg=err_msg,
-                                               session=session)
-
-                request_core.add_monitor_message(req, response, session=session)
-        else:
-            __set_transfer_state(external_host, transfer_id, state, session=session)
+    if state == RequestState.LOST:
```

Comment:
Why this change is needed ? 
Are you still working on the PR? is it a draft?

Reply:
I am not sure what happened here
This clearly doesnt need to change

Reply:
I am still working on this PR yes, but not all my time  at the moment
And now I see my PR is failing on the Multi_VO tests but for GLOBUS, which I need to investigate

Reply:
Then it's probably a good idea to convert it to a draft.